tests/pipeline/test_technical.py

```diff
@@ -138,3 +141,20 @@ def test_bollinger_bands_output_ordering(self):
         self.assertIs(lower, bbands.lower)
         self.assertIs(middle, bbands.middle)
         self.assertIs(upper, bbands.upper)
+
+
+class TestLinearWeightedMovingAverage(ZiplineTestCase):
+    def test_wma(self):
+        wma = LinearWeightedMovingAverage(
+            inputs=(USEquityPricing.close,),
+            window_length=10
+        )
+
+        today = pd.Timestamp('2014')
+        assets = np.arange(5, dtype=np.int64)
+
+        data = np.ones((10, 5))
```

Comment:
I think the constant test is good, but can we add a test with something like arange, or some handcrafted inputs?